external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaContinuousSourceSuite.scala

```diff
@@ -87,6 +87,12 @@ class KafkaContinuousSourceTopicDeletionSuite extends KafkaContinuousTest {
 
 class KafkaContinuousSourceStressForDontFailOnDataLossSuite
     extends KafkaSourceStressForDontFailOnDataLossSuite {
+
+  override def createSparkSession(): TestSparkSession = {
+    // Set maxRetries to 3 to handle NPE from `poll` when deleting a topic
+    new TestSparkSession(new SparkContext("local[10,3]", "test-sql-context", sparkConf))
```

Comment:
This looks good.

Reply:
Ur, sorry for this. While you are here, can we use more specific and different names instead `test-sql-context` for both suites?
- `KafkaSourceStressForDontFailOnDataLossSuite`
- `KafkaContinuousSourceStressForDontFailOnDataLossSuite`